publication-commons/src/main/resources/features/publish_publication.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Publish a Publication
+
+  Scenario: The User publishes a Publication
+    Given the User wants to publish a Publication
+    When they set the Accept header to "application/json"
+    And they set the Authentication header to a Bearer token with their <credentials>
```

Comment:
This makes the scenario a scenario outline, the example table is missing

Reply:
Example credentials are useless I think. I will change it.